Build/scripts/init_build.ps1

```diff
@@ -46,7 +46,7 @@ $remote = $remote.replace("mshttps", "https")
 # Get the pushId and push date time to use that for build number and build date time
 $uri = ("{0}/commits/{1}?api-version=1.0" -f $remote, $commitHash)
```

Comment:
powershell's variables are case-insensitive. Unfortunately that makes it possible to be inconsistent. I don't want to get into a huge code cleanup at this time but there's another round of changes coming after this and that might be a good time.

---
In reply to: [63264018](https://github.com/Microsoft/ChakraCore/pull/981#discussion_r63264018) [](ancestors = 63264018)